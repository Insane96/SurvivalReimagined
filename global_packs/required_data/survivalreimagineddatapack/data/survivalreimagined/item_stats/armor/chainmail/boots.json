{
	"item": "minecraft:chainmail_boots",
	"durability": 36,
	"armor": 2,
	"modifiers": [

	]
}