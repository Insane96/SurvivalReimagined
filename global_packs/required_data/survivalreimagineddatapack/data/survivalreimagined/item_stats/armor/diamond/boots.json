{
	"item": "minecraft:diamond_boots",
	"durability": 195,
	"armor": 3,
	"modifiers": [

	]
}