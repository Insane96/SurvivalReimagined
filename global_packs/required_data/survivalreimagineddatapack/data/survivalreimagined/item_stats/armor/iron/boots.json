{
	"item": "minecraft:iron_boots",
	"durability": 84,
	"armor": 2,
	"modifiers": [

	]
}