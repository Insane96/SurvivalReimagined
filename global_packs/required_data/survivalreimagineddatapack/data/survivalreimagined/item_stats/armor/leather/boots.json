{
	"item": "minecraft:leather_boots",
	"durability": 60,
	"armor": 1,
	"modifiers": [

	]
}