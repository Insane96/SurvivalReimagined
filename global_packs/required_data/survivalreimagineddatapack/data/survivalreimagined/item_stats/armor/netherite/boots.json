{
	"item": "minecraft:netherite_boots",
	"durability": 180,
	"armor": 3,
	"modifiers": [

	]
}