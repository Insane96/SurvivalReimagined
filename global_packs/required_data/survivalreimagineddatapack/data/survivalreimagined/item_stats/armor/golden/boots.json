{
	"item": "minecraft:golden_boots",
	"durability": 48,
	"armor": 1,
	"modifiers": [

	]
}